src/index.ts

```diff
@@ -31,43 +37,47 @@ const main = async() => {
 
   console.log(
     chalk.magentaBright(
-      figlet.textSync(`zkSync ${option}`, { horizontalLayout: 'full' })
+      figlet.textSync(`zkSync ${option}`, { horizontalLayout: "full" })
     )
   );
 
-  const zeekFlag = Boolean(process.argv.filter(arg => arg === "--zeek")[0]);
-  const l1RpcUrl = String(process.argv
-    .filter(arg => arg.startsWith("l1-rpc-url"))
-    .map(arg => arg.split('=')[1])[0]);
-  const l2RpcUrl = String(process.argv
-    .filter(arg => arg.startsWith("l2-rpc-url"))
-    .map(arg => arg.split('=')[1])[0]);
+  const zeekFlag = Boolean(process.argv.filter((arg) => arg === "--zeek")[0]);
+  const l1RpcUrl = String(
+    process.argv
+      .filter((arg) => arg.startsWith("l1-rpc-url"))
+      .map((arg) => arg.split("=")[1])[0]
+  );
+  const l2RpcUrl = String(
+    process.argv
+      .filter((arg) => arg.startsWith("l2-rpc-url"))
+      .map((arg) => arg.split("=")[1])[0]
+  );
 
   switch (option) {
-    case 'create':
+    case "create":
       // arg 3 is the project name
-      const projectName = process.argv[3] || '.';
+      const projectName = process.argv[3] || ".";
       await create(projectName, zeekFlag);
       break;
-    case 'deposit':
+    case "deposit":
       await deposit(zeekFlag, l1RpcUrl, l2RpcUrl);
       break;
-    case 'withdraw':
+    case "withdraw":
       await withdraw(zeekFlag, l1RpcUrl, l2RpcUrl);
       break;
-    case 'confirm-withdrawal':
+    case "confirm-withdraw":
       await confirmWithdrawal(zeekFlag, l1RpcUrl, l2RpcUrl);
```

Comment:
Since we updated to `confirmWithdraw` above, this will also need to be updated
```suggestion
      await confirmWithdraw(zeekFlag, l1RpcUrl, l2RpcUrl);
```